packages/maskbook/src/plugins/RedPacket/messages.ts

```diff
@@ -1,6 +1,12 @@
-import { RedPacketPluginID } from './constants'
+import { RedPacketPluginID, RedPacketNftPluginID } from './constants'
 import { createPluginMessage, createPluginRPC } from '@masknet/plugin-infra'
 
 if (import.meta.webpackHot) import.meta.webpackHot.accept()
 const RedPacketMessage = createPluginMessage(RedPacketPluginID)
+const RedPacketNftMessage = createPluginMessage(RedPacketNftPluginID)
```

Comment:
Please do not create multiple plugin messages. You should reuse the previous one.